cogs/ReactForRole.py

```diff
@@ -1049,7 +1050,8 @@ def add_rfr_message_emoji_role(self, emoji_role_id: int, emoji_raw: str, role_id
         :return:
         """
         self.database_manager.db_execute_commit(
-            f"""INSERT INTO RFRMessageEmojiRoles (emoji_role_id, emoji_raw, role_id) VALUES ({emoji_role_id}, \"{emoji_raw}\", {role_id});""")
+            "INSERT INTO RFRMessageEmojiRoles (emoji_role_id, emoji_raw, role_id) VALUES (?, \"?\", ?);",
```

Comment:
Don't need the \" \" around the ? when doing this, it automatically uses the data type provided (if you want to pass in an int, make sure it is an int before sending it)
```suggestion
            "INSERT INTO RFRMessageEmojiRoles (emoji_role_id, emoji_raw, role_id) VALUES (?, ?, ?);",
```